cartoframes/data/observatory/repository/category_repo.py

```diff
@@ -0,0 +1,41 @@
+from cartoframes.data.observatory.repository.repo_client import RepoClient
+
+
+def get_category_repo():
+    return CategoryRepository()
+
+
+class CategoryRepository(object):
+
+    def __init__(self):
+        self.client = RepoClient()
+
+    def get_all(self):
+        return self._to_categories(self.client.get_categories())
+
+    def get_by_id(self, category_id):
+        result = self.client.get_categories('id', category_id)
+
+        if len(result) == 0:
+            return None
+
+        return self._to_category(result[0])
+
+    def get_by_country(self, iso_code):
+        # TODO
+        pass
+
+    @staticmethod
+    def _to_category(result):
+        from cartoframes.data.observatory.category import Category
+
+        return Category({
+            'id': result['id'],
+            'name': result['name']
+        })
```

Comment:
Would `return Category(result)` work in this case? I'm thinking on minimizing the places we have to change code every time a new attribute is in the `result` dictionary.

UPDATE: OK I've seen we are mapping the data model to our Python classes. 

I'm not sure if we should do this kind of things.

For me data model and `Python` classes should be 1:1.

But on the other hand, with this approach a change in a column in the data model won't affect the Python classes, thus the notebooks using the catalog would keep working.

Opinions about this? @oleurud @esloho @arredond

Reply:
Mapping the classes, we could have a problem with previous versions in the future: if we release a version doing the mapping, and after that, we change a column from the database, the released version will fail.

As we want to have all the catalog public, we could put the responsibility of the API properties in the backend and making the CARTOframes client transparent

Reply:
Yes, I reached the same conclusion yesterday evening talking with @arredond  and already started the changes in the next issue. 

It was an easy change so I added here already.

Reply:
👍